packages/documentation-site/patternfly-docs/content/developer-resources/accessibility-testing.md

```diff
@@ -3,83 +3,79 @@ id: Testing your accessibility
+    - Text color against background color ([Understanding WCAG 1.4.3](//www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html))
+    - Text color against link color ([Technique G183](//www.w3.org/TR/WCAG20-TECHS/G183.html))
+    - Visible boundaries of buttons and form elements against adjacent background color ([Understanding WCAG 1.4.11](//www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html))
 
+## Accessibility testing checklist
 
-<br />
+We recommend referencing the following checklist as you assess your product's accessibility.
 
-### Checklist for testing
-These are some of the main areas we check for on the PatternFly team. We recommend that you check for these same areas in your applications, too. 
+These are some of the main areas we check for on the PatternFly team to ensure that components meet consistent accessibility standards, but, to check your complete implementation, we recommend checking these same areas in your products. 
 
-**General questions**
+### General criteria
 
 <label><input type="checkbox" /> VO Rotor Navigation can discover all information. </label><br/>
 <label><input type="checkbox" /> VO Shortcut Navigation can discover all information. </label><br/>
-<label><input type="checkbox" /> VO Cursor Navigation can discover all information (dependent on component and use case- easy way to navigate to component). </label><br/>
+<label><input type="checkbox" /> VO Cursor Navigation can discover all information (dependent on component and use case — there should be an easy way to navigate to the component). </label><br/>
 <label><input type="checkbox" /> Tabbed Keyboard Navigation can discover all information. </label>
-- However, if content should be hidden from a screen reader, for instance, if it’s offscreen or just presentational, make sure that content is set to aria-hidden=”true”.  <br/>
+- Unless content should be hidden from a screen reader (such as offscreen or presentational information) make sure that content is set to aria-hidden=”true”.  <br/>
 
-<label><input type="checkbox" /> Is it understandable and usable? </label><br/>
-  - <label><input type="checkbox" /> When you navigate to an item by keyboard or by screen reader, you can easily understand and use the item. </label><br/>
+<label><input type="checkbox" /> Items are understandable and usable. </label><br/>
+  - <label><input type="checkbox" /> When you navigate to an item by keyboard or by screen reader, you should be able to easily understand and use the item. </label><br/>
   - <label><input type="checkbox" /> The flow of information makes sense when navigating. Screen readers navigate the page in DOM order. If you used CSS to visually reposition elements, they might be announced in a nonsensical sequence. Instead, if you need something to appear earlier in the page, try to physically move it earlier in the DOM.</label>
 
-**Accessibility points** <br />
+### Accessibility points
```

Comment:
I'm not really sure what they could be updated to, but I feel like one or both of the headings for these sections of checklists could be updated from "General criteria" and "Accessibility points". Maybe just this section, as the entire page is regarding accessibility points technically?

Also not sure whether the current/previous content here still fits or if it's either duplicate from the "general criteria" or needs a little more information. The "Links" item in the Structure section, for example, I'm not sure of the purpose there since we mention them again in the Labels section. @jessiehuff wdyt?

Reply:
I agree on this! I can't immediately tell the difference between the lists, which pushes me towards a heading change or combining the two lists. But I'm not entirely sure what the common thread between the "accessibility points" section is, so I'm not sure how to go about renaming

Reply:
Definitely open to different names! My original intention was that the first section was thinking about the different ways of checking things (i.e. screen reader and it's different ways of navigating and keyboard) and that essentially everything should work through each of the general a11y questions and thinking big picture (i.e. just like the questions asked in the development section - is it discoverable, able to be interacted with, and understandable). The second section was more getting into the nitty gritty of checking specific things, starting from high level like structure to more granular things like labels. So totally open to anything that communicates that better! If both of you were confused, it definitely sounds like something should be changed!

In terms of the links in both the structure and labels section because when I'm testing structure, especially using the rotor menu, I check each of those major elements (headings, landmarks, links, form controls, tables) and make sure that nothing immediately looks off with the way that they're handled. That's probably not totally clear though. 🙃 I guess that's what happens when a checklist starts off as more of something for your personal use to trying to communicate it more publicly haha.